src/renderer/components/NewTeamModal.tsx

```diff
@@ -115,9 +123,17 @@ export default class NewTeamModal extends React.PureComponent<Props, State> {
     }
 
     handleTeamUrlChange = (e: React.ChangeEvent<HTMLInputElement>) => {
-        this.setState({
-            teamUrl: e.target.value,
-        });
+        const teamUrl = e.target.value;
+        this.setState({teamUrl});
+        if (!teamUrl.startsWith('http') && teamUrl.length >= 4) {
```

Comment:
Technically `a.b` could be a valid domain, so we may not want to bother enforcing the length requirement.